{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/branch/3.15-community/vuln-orphaned","items":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeMatch":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeUri":"cpe:2.3:a:libdwarf_project:libdwarf:*:*:*:*:*:*:*:*","id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#cpeMatch/320737","maximumVersion":null,"maximumVersionOp":"==","minimumVersion":null,"minimumVersionOp":"==","package":"https://security.alpinelinux.org/srcpkg/libdwarf","type":"CPEMatch","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"}],"cvss3":{"score":6.5,"vector":"CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"},"description":"dwarf_leb.c in libdwarf allows attackers to cause a denial of service (SIGSEGV).","id":"https://security.alpinelinux.org/vuln/CVE-2015-8538","ref":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#ref/257229","referenceType":"Issue Tracking","rel":"https://bugzilla.redhat.com/show_bug.cgi?id=1291299","type":"Reference"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#ref/257230","referenceType":"Mailing List","rel":"http://www.openwall.com/lists/oss-security/2015/12/10/3","type":"Reference"}],"state":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/20617","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/20210528-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/20621","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/20201020-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/27584","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/20210528-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/33273","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/20210528-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/43744","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/20210528-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/45293","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/20210528-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/51770","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/20210528-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/52597","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/0.5.0-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/53634","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/0.6.0-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/53699","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/0.6.0-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/54575","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/0.6.0-r2","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/56957","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/0.6.0-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/62975","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/0.6.0-r2","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2015-8538#state/63523","packageVersion":"https://security.alpinelinux.org/srcpkg/libdwarf/0.7.0-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2015-8538"}],"type":"Vulnerability"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeMatch":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeUri":"cpe:2.3:a:dena:h2o:*:*:*:*:*:*:*:*","id":"https://security.alpinelinux.org/vuln/CVE-2021-43848#cpeMatch/349959","maximumVersion":"2021-12-20","maximumVersionOp":"<","minimumVersion":null,"minimumVersionOp":">=","package":"https://security.alpinelinux.org/srcpkg/h2o","type":"CPEMatch","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-43848"}],"cvss3":{"score":5.9,"vector":"CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:N/A:N"},"description":"h2o is an open source http server. In code prior to the `8c0eca3` commit h2o may attempt to access uninitialized memory. When receiving QUIC frames in certain order, HTTP/3 server-side implementation of h2o can be misguided to treat uninitialized memory as HTTP/3 frames that have been received. When h2o is used as a reverse proxy, an attacker can abuse this vulnerability to send internal state of h2o to backend servers controlled by the attacker or third party. Also, if there is an HTTP endpoint that reflects the traffic sent from the client, an attacker can use that reflector to obtain internal state of h2o. This internal state includes traffic of other connections in unencrypted form and TLS session tickets. This vulnerability exists in h2o server with HTTP/3 support, between commit 93af138 and d1f0f65. None of the released versions of h2o are affected by this vulnerability. There are no known workarounds. Users of unreleased versions of h2o using HTTP/3 are advised to upgrade immediately.","id":"https://security.alpinelinux.org/vuln/CVE-2021-43848","ref":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2021-43848#ref/282069","referenceType":"MISC","rel":"https://github.com/h2o/h2o/commit/8c0eca3","type":"Reference"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2021-43848#ref/282070","referenceType":"CONFIRM","rel":"https://github.com/h2o/h2o/security/advisories/GHSA-f9xw-j925-m4m4","type":"Reference"}],"state":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-43848#state/29398","packageVersion":"https://security.alpinelinux.org/srcpkg/h2o/2.2.6-r8","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-43848"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-43848#state/29399","packageVersion":"https://security.alpinelinux.org/srcpkg/h2o/2.2.6-r8","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-43848"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-43848#state/33317","packageVersion":"https://security.alpinelinux.org/srcpkg/h2o/2.2.6-r8","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-43848"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-43848#state/41371","packageVersion":"https://security.alpinelinux.org/srcpkg/h2o/2.2.6-r9","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-43848"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-43848#state/45330","packageVersion":"https://security.alpinelinux.org/srcpkg/h2o/2.2.6-r9","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-43848"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-43848#state/63112","packageVersion":"https://security.alpinelinux.org/srcpkg/h2o/2.2.6-r9","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-43848"}],"type":"Vulnerability"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeMatch":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeUri":"cpe:2.3:a:tinyxml_project:tinyxml:*:*:*:*:*:*:*:*","id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#cpeMatch/326365","maximumVersion":"2.6.2","maximumVersionOp":"<=","minimumVersion":null,"minimumVersionOp":">=","package":"https://security.alpinelinux.org/srcpkg/tinyxml","type":"CPEMatch","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeUri":"cpe:2.3:a:tinyxml_project:tinyxml:2.3.0:beta:*:*:*:*:*:*","id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#cpeMatch/345304","maximumVersion":"2.3.0","maximumVersionOp":"==","minimumVersion":null,"minimumVersionOp":"==","package":"https://security.alpinelinux.org/srcpkg/tinyxml","type":"CPEMatch","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeUri":"cpe:2.3:a:tinyxml_project:tinyxml:2.3.1:beta:*:*:*:*:*:*","id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#cpeMatch/345305","maximumVersion":"2.3.1","maximumVersionOp":"==","minimumVersion":null,"minimumVersionOp":"==","package":"https://security.alpinelinux.org/srcpkg/tinyxml","type":"CPEMatch","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeUri":"cpe:2.3:a:tinyxml_project:tinyxml:*:*:*:*:*:*:*:*","id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#cpeMatch/345306","maximumVersion":"2.6.2","maximumVersionOp":"<=","minimumVersion":"2.3.2","minimumVersionOp":">=","package":"https://security.alpinelinux.org/srcpkg/tinyxml","type":"CPEMatch","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"}],"cvss3":{"score":7.5,"vector":"CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"},"description":"TinyXML through 2.6.2 has an infinite loop in TiXmlParsingData::Stamp in tinyxmlparser.cpp via the TIXML_UTF_LEAD_0 case. It can be triggered by a crafted XML message and leads to a denial of service.","id":"https://security.alpinelinux.org/vuln/CVE-2021-42260","ref":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#ref/265956","referenceType":"MISC","rel":"https://sourceforge.net/p/tinyxml/bugs/141/","type":"Reference"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#ref/297250","referenceType":"Mailing List","rel":"https://lists.debian.org/debian-lts-announce/2022/04/msg00019.html","type":"Reference"}],"state":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#state/21850","packageVersion":"https://security.alpinelinux.org/srcpkg/tinyxml/2.6.2-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#state/21851","packageVersion":"https://security.alpinelinux.org/srcpkg/tinyxml/2.6.2-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#state/21852","packageVersion":"https://security.alpinelinux.org/srcpkg/tinyxml/2.6.2-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#state/27803","packageVersion":"https://security.alpinelinux.org/srcpkg/tinyxml/2.6.2-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#state/33267","packageVersion":"https://security.alpinelinux.org/srcpkg/tinyxml/2.6.2-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#state/45338","packageVersion":"https://security.alpinelinux.org/srcpkg/tinyxml/2.6.2-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2021-42260#state/63472","packageVersion":"https://security.alpinelinux.org/srcpkg/tinyxml/2.6.2-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2021-42260"}],"type":"Vulnerability"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeMatch":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeUri":"cpe:2.3:a:xpdfreader:xpdf:4.03:*:*:*:*:*:*:*","id":"https://security.alpinelinux.org/vuln/CVE-2022-27135#cpeMatch/380244","maximumVersion":"4.03","maximumVersionOp":"==","minimumVersion":null,"minimumVersionOp":"==","package":"https://security.alpinelinux.org/srcpkg/xpdf","type":"CPEMatch","vuln":"https://security.alpinelinux.org/vuln/CVE-2022-27135"}],"cvss3":{"score":5.5,"vector":"CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"},"description":"xpdf 4.03 has heap buffer overflow in the function readXRefTable located in XRef.cc. An attacker can exploit this bug to cause a Denial of Service (Segmentation fault) or other unspecified effects by sending a crafted PDF file to the pdftoppm binary.","id":"https://security.alpinelinux.org/vuln/CVE-2022-27135","ref":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2022-27135#ref/294309","referenceType":"MISC","rel":"https://github.com/verf1sh/Poc/blob/master/poc_ppm","type":"Reference"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2022-27135#ref/294310","referenceType":"MISC","rel":"https://forum.xpdfreader.com/viewtopic.php?f=3&t=42232","type":"Reference"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2022-27135#ref/294311","referenceType":"MISC","rel":"https://github.com/verf1sh/Poc/blob/master/pic_ppm.png","type":"Reference"}],"state":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2022-27135#state/32677","packageVersion":"https://security.alpinelinux.org/srcpkg/xpdf/4.03-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2022-27135"}],"type":"Vulnerability"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeMatch":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeUri":"cpe:2.3:a:eficode:influxdb:*:*:*:*:*:jenkins:*:*","id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#cpeMatch/87713","maximumVersion":"1.21","maximumVersionOp":"<=","minimumVersion":null,"minimumVersionOp":">=","package":"https://security.alpinelinux.org/srcpkg/influxdb","type":"CPEMatch","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"}],"cvss3":{"score":8.8,"vector":"CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"},"description":"Jenkins InfluxDB Plugin 1.21 and earlier stored credentials unencrypted in its global configuration file on the Jenkins master where they can be viewed by users with access to the master file system.","id":"https://security.alpinelinux.org/vuln/CVE-2019-10329","ref":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#ref/55620","referenceType":"Vendor Advisory","rel":"https://jenkins.io/security/advisory/2019-05-31/#SECURITY-1403","type":"Reference"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#ref/55621","referenceType":"Mailing List","rel":"http://www.openwall.com/lists/oss-security/2019/05/31/2","type":"Reference"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#ref/55622","referenceType":"Third Party Advisory","rel":"http://www.securityfocus.com/bid/108540","type":"Reference"}],"state":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/17530","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.3-r3","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/17668","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.3-r3","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/21095","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.3-r4","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/21145","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.3-r4","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/27804","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.3-r4","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/28899","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/29220","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r1","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/30687","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r2","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/31375","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r3","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/31424","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.3-r5","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/32461","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r4","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/32502","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.3-r6","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/33272","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r4","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/40861","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r5","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/40874","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r5","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/41274","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r6","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/41366","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r6","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/42521","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r7","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/42534","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r7","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/42965","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r8","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/42988","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r8","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/43932","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r9","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/44985","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r9","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/51920","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r10","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/52095","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r10","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/53274","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r11","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/53517","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r12","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/53523","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r11","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/54454","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r13","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/54534","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r12","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2019-10329#state/55544","packageVersion":"https://security.alpinelinux.org/srcpkg/influxdb/1.8.10-r13","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2019-10329"}],"type":"Vulnerability"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeMatch":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","cpeUri":"cpe:2.3:a:gogs:gogs:*:*:*:*:*:*:*:*","id":"https://security.alpinelinux.org/vuln/CVE-2022-1464#cpeMatch/375646","maximumVersion":"0.12.7","maximumVersionOp":"<","minimumVersion":null,"minimumVersionOp":">=","package":"https://security.alpinelinux.org/srcpkg/gogs","type":"CPEMatch","vuln":"https://security.alpinelinux.org/vuln/CVE-2022-1464"}],"cvss3":{"score":5.4,"vector":"CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:C/C:L/I:L/A:N"},"description":"Stored xss bug in GitHub repository gogs/gogs prior to 0.12.7. As the repo is public , any user can view the report and when open the attachment then xss is executed. This bug allow executed any javascript code in victim account .","id":"https://security.alpinelinux.org/vuln/CVE-2022-1464","ref":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2022-1464#ref/295271","referenceType":"MISC","rel":"https://github.com/gogs/gogs/commit/bc77440b301ac8780698be91dff1ac33b7cee850","type":"Reference"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","id":"https://security.alpinelinux.org/vuln/CVE-2022-1464#ref/295272","referenceType":"CONFIRM","rel":"https://huntr.dev/bounties/34a12146-3a5d-4efc-a0f8-7a3ae04b198d","type":"Reference"}],"state":[{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2022-1464#state/32559","packageVersion":"https://security.alpinelinux.org/srcpkg/gogs/0.12.6-r2","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2022-1464"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":false,"id":"https://security.alpinelinux.org/vuln/CVE-2022-1464#state/32560","packageVersion":"https://security.alpinelinux.org/srcpkg/gogs/0.12.6-r2","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2022-1464"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2022-1464#state/32579","packageVersion":"https://security.alpinelinux.org/srcpkg/gogs/0.12.7-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2022-1464"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2022-1464#state/36641","packageVersion":"https://security.alpinelinux.org/srcpkg/gogs/0.12.7-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2022-1464"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2022-1464#state/49091","packageVersion":"https://security.alpinelinux.org/srcpkg/gogs/0.12.7-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2022-1464"},{"@context":"https://security.alpinelinux.org/static/context.jsonld","fixed":true,"id":"https://security.alpinelinux.org/vuln/CVE-2022-1464#state/60004","packageVersion":"https://security.alpinelinux.org/srcpkg/gogs/0.12.7-r0","type":"VulnerabilityState","vuln":"https://security.alpinelinux.org/vuln/CVE-2022-1464"}],"type":"Vulnerability"}],"type":"Collection"}
